install.sh

```diff
@@ -7,8 +7,8 @@ cd $(dirname $0)
 # use correct node version and install dependencies
 nvm install
 nvm use
-npm i -g yarn
-yarn install
+npm i -g pnpm@7
```

Comment:
PNPMv8 was released two days ago. Might it be worth it, to directly target that version? (see https://github.com/pnpm/pnpm/releases)

Reply:
done